tooling-mp-jwt-auth-tool/pom.xml

```diff
@@ -15,8 +15,19 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
+            <!-- TODO: remove once all microprofile specs will be updated to MP4 -->
+            <version>4.0.1</version>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>org.bitbucket.b_c</groupId>
+            <artifactId>jose4j</artifactId>
```

Comment:
`<scope>test</scope>`?

Reply:
used `<scope>provided</scope>` because `jose4j` is used in e.g. JweHelper